pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -10,7 +10,7 @@
 , qtwebengine
 , enableWideVine ? false
 , widevine-cdm
-, enableVulkan ? stdenv.isLinux
+, enableVulkan ? false
```

Comment:
I think it would be good to mention the issue in a comment. (Assuming we want to turn Vulkan on again once the issue is fixed)

Reply:
I thought about it, but since I cannot reproduce the issue on my system, wasn't sure what exactly to put there. Seems its only particular AMD chips that are affected, by that's just a guess.

Reply:
I'm also experiencing problems on i915 on Alder Lake (Framework 12th Gen Intel)

Maybe a comment could just say "Crashes or has responsiveness issues on multiple graphics vendors (2023-11-05)"?

There may be an upstream Qt bug about it that could be referenced as well